packages/suite-web/e2e/tests/onboarding/analytics-consent.test.ts

```diff
@@ -89,20 +73,6 @@ describe('Onboarding - analytics consent', () => {
         cy.getTestElement('@settings/menu/close').should('be.visible');
     });
 
-    it('shows analytics consent when going to settings and back on initialized T2B1 device', () => {
```

Comment:
this analytics-consent test is written in unit-test way. this is not right, and if we wanted to test every scenario with every device we would soon explode in complexity. removing model R variants